po/uk.po

```diff
@@ -437,11 +437,15 @@ msgid "Halt the system while an application is inhibiting this"
 msgstr "Зупинити систему, коли програми намагаються перешкодити цьому"
 
 #: src/login/org.freedesktop.login1.policy:247
+#| msgid ""
+#| "Authentication is required to hibernate the system while an application "
+#| "is inhibiting this."
```

Comment:
Please remove the comment.

Reply:
This is automated gettext fuzzy matching. Removed.